gmso/tests/test_potential_templates.py

```diff
@@ -0,0 +1,129 @@
+import os
+import glob
+import pytest
+
+import sympy
+
+from gmso.lib.potential_templates import PotentialTemplates, JSON_DIR
+from gmso.exceptions import GMSOError
+from gmso.tests.base_test import BaseTest
+
+
+class TestPotentialTemplates(BaseTest):
+    @pytest.fixture
+    def templates(self):
+        return PotentialTemplates()
+
+    def test_singleton_behavior(self, templates):
+        assert id(templates) == id(PotentialTemplates())
+        assert id(PotentialTemplates()) == id(PotentialTemplates())
+
+    def test_lennard_jones_potential(self, templates):
```

Comment:
This and the next 10 or so tests are effectively SIMD, would it be useful to wrap them up into a single test?

Reply:
This is done because of aiding future contributors with a contributing guideline. Other Implementations could trim the test, I agree.